packages/terra-tabs/src/clinical-lowlight-theme/Tab.module.scss

```
@@ -1,6 +1,13 @@
 @import '~terra-mixins/lib/Mixins';
 
 :local {
+  .button:focus {
+    &[data-terra-tabs-show-focus-styles='true'] {
+      outline: var(--terra-tabs-add-button-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-tabs-add-button-focus-outline-offset, -5px);
```

Can we have this as -2px to keep the outline consistent with other components. 
```suggestion
      outline-offset: var(--terra-tabs-add-button-focus-outline-offset, -2px);
```

---

Done https://github.com/cerner/terra-framework/commit/09f89f4169db28da2af3774ea6bd197df5344813